[2025-04-01T15:52:47.123+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.load scheduled__2025-04-01T14:45:00+00:00 [queued]>
[2025-04-01T15:52:47.129+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.load scheduled__2025-04-01T14:45:00+00:00 [queued]>
[2025-04-01T15:52:47.129+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-04-01T15:52:47.139+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load> on 2025-04-01 14:45:00+00:00
[2025-04-01T15:52:47.143+0000] {standard_task_runner.py:57} INFO - Started process 804 to run task
[2025-04-01T15:52:47.145+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_pipeline', 'load', 'scheduled__2025-04-01T14:45:00+00:00', '--job-id', '231', '--raw', '--subdir', 'DAGS_FOLDER/etl_dags.py', '--cfg-path', '/tmp/tmp9kfitb2i']
[2025-04-01T15:52:47.147+0000] {standard_task_runner.py:85} INFO - Job 231: Subtask load
[2025-04-01T15:52:47.159+0000] {logging_mixin.py:154} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-01T15:52:47.183+0000] {task_command.py:416} INFO - Running <TaskInstance: etl_pipeline.load scheduled__2025-04-01T14:45:00+00:00 [running]> on host f263ca9a5772
[2025-04-01T15:52:47.233+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T14:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-01T14:45:00+00:00'
[2025-04-01T15:53:13.390+0000] {logging_mixin.py:154} INFO - 4 nguồn dữ liệu đã load xong, bắt đầu load AQI...
[2025-04-01T15:53:24.147+0000] {logging_mixin.py:154} INFO - Load hoàn tất ❤️
[2025-04-01T15:53:24.646+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-01T15:53:24.653+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=load, execution_date=20250401T144500, start_date=20250401T155247, end_date=20250401T155324
[2025-04-01T15:53:24.666+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-01T15:53:24.681+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-01T16:33:51.335+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.load scheduled__2025-04-01T14:45:00+00:00 [queued]>
[2025-04-01T16:33:52.819+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.load scheduled__2025-04-01T14:45:00+00:00 [queued]>
[2025-04-01T16:33:52.819+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-04-01T16:33:54.482+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load> on 2025-04-01 14:45:00+00:00
[2025-04-01T16:33:54.487+0000] {standard_task_runner.py:57} INFO - Started process 218 to run task
[2025-04-01T16:33:54.489+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_pipeline', 'load', 'scheduled__2025-04-01T14:45:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/etl_dags.py', '--cfg-path', '/tmp/tmp2rl8pji8']
[2025-04-01T16:33:54.490+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask load
[2025-04-01T16:34:02.142+0000] {task_command.py:416} INFO - Running <TaskInstance: etl_pipeline.load scheduled__2025-04-01T14:45:00+00:00 [running]> on host 9a57a1922f75
[2025-04-01T16:34:08.000+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T14:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-01T14:45:00+00:00'
[2025-04-01T16:34:16.640+0000] {logging_mixin.py:154} INFO - 4 nguồn dữ liệu đã load xong, bắt đầu load AQI...
[2025-04-01T16:34:19.291+0000] {logging_mixin.py:154} INFO - Load hoàn tất ❤️
[2025-04-01T16:34:19.427+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-01T16:34:20.622+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=load, execution_date=20250401T144500, start_date=20250401T163351, end_date=20250401T163420
[2025-04-01T16:34:21.281+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-01T16:34:23.962+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
